cmd/create/accountroles/cmd.go

```diff
@@ -358,120 +360,144 @@ func run(cmd *cobra.Command, argv []string) {
 	}
 }
 
-func buildCommands(prefix string, permissionsBoundary string, accountID string, defaultPolicyVersion string,
-	path string, managedPolicies bool, policies map[string]*cmv1.AWSSTSPolicy) (string, error) {
-	commands := []string{}
+type creator interface {
+	createRoles(*rosa.Runtime, rolesCreationInput) error
+	getRoleTags(string, rolesCreationInput) map[string]string
+	buildCommands(rolesCreationInput) (string, error)
+}
+
+func initCreator(managedPolicies bool) creator {
+	if managedPolicies {
+		return managedPoliciesCreator{}
```

Comment:
should be `&managedPoliciesCreator`.

An interface is a pointer so implementing structs should also be pointers

Reply:
Makes sense, fixed, thanks.